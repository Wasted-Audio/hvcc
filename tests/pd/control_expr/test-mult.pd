#N canvas 670 156 450 300 12;
#X obj 55 17 loadbang;
#X obj 55 152 print;
#X msg 109 77 3.1;
#X msg 55 77 2.3;
#X obj 55 119 expr $f1*$f2*$i3;
#X obj 55 42 t b b b;
#X msg 164 77 5.5;
#X connect 0 0 5 0;
#X connect 2 0 4 1;
#X connect 3 0 4 0;
#X connect 4 0 1 0;
#X connect 5 0 3 0;
#X connect 5 1 2 0;
#X connect 5 2 6 0;
#X connect 6 0 4 2;
