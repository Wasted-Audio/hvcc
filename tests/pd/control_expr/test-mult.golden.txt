[@ 0.000] print: 35.65
